docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: post
+title: Let's Make A Watch Gadget!
+---
+
+# Let's Make A Watch Gadget!
+
+This tutorial is intended to follow the [first one](https://aardvarkxr.github.io/aardvark/getting_started/), where you've learned about the prerequisites for developing AardvarkXR gadgets, gadget initialization, and some scene graph anatomy. If you haven't worked through that tutorial yet, please do so and come back when you're ready for more - we'll be building on that information and assuming you've mastered the basics.
+
+In this guide we'll be building a simple digital watch; we'll utilize panels and 3D models and implement a a little interactivity. Once you've worked through this guide, you should have two things:
+
+* A vintage, functioning 90s G-Shock watch gadget 
+* A better understanding of the Aardvark scene graph, familiarity with useful nodes, and the gadget development workflow.
+
+# Step 1: _The First Time_ - Tooling and debugging
+
+If you've worked on wearable VR/AR experiences, you're already familiar with the ergonomic difficulties of iterating on development and alternating between coding (headset off) and testing (headset on). You might already have a workflow you're comfortable with that mitigates this, in which case feel free to skim or skip this section. In any case, here's my typical workflow in the hope that it might be of use to you too:
+
+1. I like to stay in VR and work in the virtual desktop to minimize headset fidgeting time. It's worth knowing there are several good VR desktop solutions available like [this one, ](https://www.vrdesktop.net/) [this one, ](https://store.steampowered.com/app/457550/Bigscreen_Beta/) [this one](https://store.steampowered.com/app/1068820/OVR_Toolkit/) and [this one](https://store.steampowered.com/app/1178460/Desktop_Portal/) that play well with SteamVR, some of which look and feel much better than the built-in one. 
+
+2. Coding is done in VS Code, with the `npm start` script running in VS Code's terminal (you can launch a new terminal instance from VS Code's Terminal tab):
+   
+![workflow-1](./media/0-0-workflow.jpg)
+
+Note that the script watches the files and recompiles on change, so you don't have to "refresh" anything aside from hitting save.
+
+2. When iterating, the Aardvark renderer does *not* have to be restarted to reflect gadget updates but you *do* have to instantiate a new gadget every time you want to see the latest compiled changes (and probably want to discard that gadget after testing).
```

Comment:
You can also attach the debugger in VS Code and reload with the debugger widget button or ctrl-shift-F5. The create script will make a launch.json that attached to avrenderer to enable this.

Reply:
added